### ISSUE TITLE
Enable R_FLAG_ZONE_USE_SDB (and fix whatever its broken)

### ISSUE THREAD

**Issue by radare** [radare/radare2#11992]
Evaluate results and consider removing this code or keep the current implementation

**Comment by radare** [comment]
ping @ret2libc

**Comment by ret2libc** [comment]
I don't think this is *that* important. I'm going to mark it for milestone 9999, flag zones are not one of the main features IMO.

**Comment by radare** [comment]
I use them pretty frequently

**Comment by trufae** [comment]
Resolved already, we dont use sdb in flag zones anymore
